contracts/interfaces/IExtendedKlerosArbitrator.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IExtendedKlerosArbitrator {
```

Comment:
So this interface is for exposing methods that are not defined in `IArbitrator` but we know that the address points to a contract that has those methods defined? How can we be sure of that?

Reply:
Yes, and we can't because these functions are not a part of the Kleros standard and I suspect that they may be abandoned in their next Kleros version (I can ask). Arguably these view functions aren't even needed and the frontend can call the arbitrator directly. I just like providing everything that will be needed under one contract that can be Multicall'ed and not require @mcoetzee to have to read Kleros contracts.
I think the best solution is to keep these in a separate convenience contract as in the DAO contracts. The main downside of this will be the frontend won't be able to Multicall values from ClaimsManagerWithKlerosArbitration and ConvenienceKlerosArbitration with a single call. This is more of a limitation of the OpenZeppelin Multicall implementation (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Multicall.sol) because it only allows `address(this)` to be called while the original Multicall allows external contracts to be called too (https://github.com/makerdao/multicall/blob/master/src/Multicall.sol).
My main plan was to use these in ClaimsManagerWithKlerosArbitration (and be able to Multicall everything conveniently) and stop using them once we start using a Kleros arbitrator that doesn't support them. It feels like this is related to this (https://github.com/api3dao/claims-manager/pull/21#discussion_r922850739) in that the Kleros proxy contract that will include these view functions may have an immutable Kleros arbitrator and a new Kleros arbitrator may require a new proxy to be implemented that will handle backwards compatibility.